docs/getting-started/ory-cloud-oauth2.mdx

```diff
@@ -1,248 +1,187 @@
 ---
-id: ory-cloud-oauth2
-title: Perform OAuth2 Authorization Code Grant with The Ory Network
+id: ory-network-oauth2
+title: Get started with common OAuth2 flows in the Ory Network
 sidebar_label: Federation with OAuth2
 ---
 
-This guide shows you a sample implementation of the
-[Authorization Code Grant](https://www.rfc-editor.org/rfc/rfc6749#section-1.3.1) OAuth2 flow using Ory OAuth2 & OpenID Connect in
-Ory Network. It takes you on a complete journey, from initial setup to running an OAuth2 flow.
+# Common OAuth2 flows in the Ory Network
 
-:::tip
-
-Ory OAuth2 & OpenID Connect (based on [Ory Hydra](../hydra/index.md)) is available in The Ory Network out of the box, which means
-you can use OIDC, Authorization Code Grant, Client Credentials Grant, and more. Read
-[this document](../hydra/concepts/before-oauth2) to learn more about OAuth2 and evaluate if it's the right fit for you.
-
-:::
+Ory OAuth2 & OpenID Connect (based on the [Ory Hydra Federation Server](../hydra/index.md)) is available in The Ory Network out of
+the box. This means that you can use OIDC, Authorization Code Grant, Client Credentials Grant, and more, without additional
+configuration or extra charge.
 
-The implementation is based on a simple Node.js application with login and consent screens that allow you to simulate a common a
-real-world scenario where an application (client) asks the user for consent to get access to certain data or operations (scopes).
+Following this guide allows you to experience the most commonly used OAuth2 flows and see how they work in Ory Network. The
+examples will take you through the following steps:
 
-For example, a service for selling goods allows users to add pictures from their Google Photos albums to their listings. To get
-access to the user's photos, the OAuth2 client of the goods selling service asks the user for consent to access data on their
-behalf. The request includes scopes - a list of operations the client wants to perform on behalf of the user. When the user
-agrees, the OAuth2 client gets an authorization code which it exchanges for an `id_token` and `access_token` pair issued by Google
-Photos. These tokens allow the client to access the user's resources from an external service.
-
-By following this guide you will:
-
-- Create an OAuth2 client in The Ory Network.
-- Run a simple login and consent Node.js application that allows the user to grant or reject access to their data on your machine.
-- Configure The Ory Network to work with the sample application that runs on your machine.
-- Perform Authorization Code Grant in The Ory Network using the login and consent app and a sample web server acting as an OAuth2
-  Client.
+- Creating OAuth2 clients in The Ory Network with the Ory CLI
+- Using the Authorization Code Grant (with OpenID) flow with user accounts created in Ory Identities and login and consent screens
```

Comment:
I would phrase it more in a way : federation to Ory Identity Server with consent screen supplied by Ory Account Experience

Reply:
done